build-logic/jvm/src/main/kotlin/gradlebuild.unittest-and-compile.gradle.kts

```diff
@@ -319,6 +320,10 @@ fun configureTests() {
                 }
             }
         }
+
+        predictiveSelection {
```

Comment:
There is a statement above about `project.enableExperimentalTestFiltering()`. I think that statement needs to be replaced by the new logic, right?